ouroboros-consensus-test/test-consensus/Test/Consensus/Mempool.hs

```diff
@@ -11,7 +11,16 @@
 {-# LANGUAGE TupleSections       #-}
 
 {-# OPTIONS_GHC -Wno-incomplete-uni-patterns #-}
-module Test.Consensus.Mempool (tests) where
+module Test.Consensus.Mempool (
+    tests
+    -- * Exported for other tests
+  , TestBlock
+  , applyTxToLedger
+  , genTxs
+  , genValidTxs
+  , testInitLedger
+  , testLedgerConfig
```

Comment:
My preferred structuring is to put common definitions in a separate module (something like `Mempool.Setup`?), such that `Mempool` and `Mempool.StateMachine` can depend on that module

Reply:
Done